components/table/hooks/useSelection.tsx

```
@@ -535,7 +535,7 @@ const useSelection = <RecordType extends AnyObject = AnyObject>(
           const key = getRowKey(record, index);
           const checked = keySet.has(key);
           const checkboxProps = checkboxPropsMap.get(key) as unknown as RadioProps;
-          const defaultAriaLabel = `Select row ${index + 1}`;
+          const defaultAriaLabel = tableLocale.selectRow?.replace('${index + 1}', `${index + 1}`) || `Select row ${index + 1}`;
```

ESLint

[error] Unexpected template string expression.

(no-template-curly-in-string)